# Taken from gentoo:
# https://gitweb.gentoo.org/repo/gentoo.git/plain/www-client/chromium/files/chromium-clang-r3.patch
# 
# Prevents this error:
# ../../device/u2f/u2f_ble_transaction.cc:134:27: error: no viable overloaded '='
#   request_cont_fragments_ = {};
#   ~~~~~~~~~~~~~~~~~~~~~~~ ^ ~~

--- a/device/u2f/u2f_ble_transaction.cc
+++ b/device/u2f/u2f_ble_transaction.cc
@@ -131,7 +131,7 @@
 
 void U2fBleTransaction::OnError() {
   request_frame_.reset();
-  request_cont_fragments_ = {};
+  request_cont_fragments_ = base::queue<U2fBleFrameContinuationFragment>();
   response_frame_assembler_.reset();
   std::move(callback_).Run(base::nullopt);
 }
